raiden/api/rest.py

```diff
@@ -594,6 +604,24 @@ def get_token_network_events(self, token_address, from_block, to_block):
         except InvalidBlockNumberInput as e:
             return api_error(str(e), status_code=HTTPStatus.CONFLICT)
 
+    def get_channel_history_events(
+            self,
+            token_address,
+            partner_address=None,
+            from_block=None,
```

Comment:
blocks won't be available for internal events.

Reply:
If we merge https://github.com/raiden-network/raiden/pull/1859 then this should be removed, but if we wait with that until the next release I'd say we can merge this PR now and fix this with https://github.com/raiden-network/raiden/pull/1859

Reply:
I would merge 1859 for this release as I mentioned in RC

> but if we wait with that until the next release I'd say we can merge this PR now and fix this with #1859

yes that would be the correct way to go about doing it if we skup 1859 for this release